### PR TITLE
[APPCOMPAT] run apps in compatibility with 6.1 SP1, 6.2 and 6.3

### PR DESCRIPTION
**Pull request by Getequ**

### CODE REVIEW

File: dll/appcompat/shims/layer/versionlie.c [hunk 1]
```diff
@@ -230,6 +231,9 @@ BOOL WINAPI SHIM_OBJ_NAME(APIHook_GetVersionExW)(LPOSVERSIONINFOEXA lpOsVersionI
 #define VERSION_INFO    g_Win7RTM
 #include "versionlie.inl"
 
+#define SHIM_NS         Win7SP1VersionLie
```
Comment (learn-more) [inline dll/appcompat/shims/layer/versionlie.c]: Does windows provide this shim?
If so, can you please add an apitest showing that this shim reports the correct values?

**Comment by MailYouLater** [comment]
I don't know what it offers in the APIs, but this is what the Compatibility tab on Windows 10 looks like:

File: dll/appcompat/shims/layer/versionlie.c (hunk 1 above)
Comment (Getequ) [reply dll/appcompat/shims/layer/versionlie.c]: >can you please add an apitest showing that this shim reports the correct values?

ok, i thought adding "run_test("Win7SP1VersionLie", &g_Win7SP1);" is enough

Comment (MailYouLater) [reply dll/appcompat/shims/layer/versionlie.c]: Can you tell what version number Windows 8 and/or 10 claim when they are set to Windows 7 compatibility mode?

Comment (learn-more) [reply dll/appcompat/shims/layer/versionlie.c]: I am seeing `WinSrv08SP1` but no `Win7SP1`

**Pull request merged**